pkgs/development/python-modules/jaxlib/default.nix

```
@@ -168,7 +169,7 @@ let
       numpy
       openssl
       pkgs.flatbuffers
-      protobuf
+      pkgs.protobuf
```

```suggestion
      protobuf
```
why is pkgs even in the inputs already?

---

good question! when attribute names in `pkgs` and `python3Packages` conflict, argumes evaluate to the `python3Packages` one. When we actually require the `pkgs` one, we have to specify as such. it looks odd at first but fwiw I am not introducing this pattern in this PR